src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/InputDesignForExtensionMultipleAnnotations.java

```diff
@@ -0,0 +1,23 @@
+package com.puppycrawl.tools.checkstyle.checks.design;
+
+import java.util.List;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class InputDesignForExtensionMultipleAnnotations {
+
+    @Ignore
+    @Deprecated
+    public void foo1() {
+        foo3();
+    }
+
+    @Deprecated
+    @Ignore
+    public void foo2() {
+        foo3();
+    }
+
+    private void foo3() {}
```

Comment:
please add test case for simple annotation usage for example "@Inject" , as you detected false-negative at https://mezk.github.io/i3830-DesignForExtension/diff/apache-struts/index.html#A2

Please add test cases for comments between annotations. 
Make test with annotations as single line with method signature.